django/core/mail/message.py

```diff
@@ -248,6 +248,11 @@ def __init__(
                 else:
                     self.attach(*attachment)
         self.extra_headers = headers or {}
+        self.headers_with_one_occurrence = {
+            "To": self.to,
+            "Reply-To": self.reply_to,
+            "Cc": self.cc,
+        }
```

Comment:
The PR contains the changes you made in https://github.com/django/django/pull/17642.
Please remove them.